### PR TITLE
API for client side requests, Getting-started page (Saturday Morn' Peer Programming)

### PR DESCRIPTION
**Pull request by djavorek**

### CODE REVIEW

File: client/src/lib/shared/proxy.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+interface RequestProxyConfig {
+	fetch: typeof fetch;
+	cookies: Cookies;
+	method: 'GET' | 'POST' | 'DELETE';
+	path: string;
+	request?: Request;
+}
+
+export const requestAsProxy = async ({
```
Comment (djavorek) [inline client/src/lib/shared/proxy.ts:13]: 🔥 Nice work

File: server/proc_macros/postman_collection.json [hunk 2]
```diff
@@ -0,0 +1,711 @@
+{
```
[the remark this answers is not in the document]
Comment (djavorek) [reply server/proc_macros/postman_collection.json:1]: Nice idea

**Comment by djavorek** [comment]
All these changes seem fine to me.
 Can we merge this now, or should we wait another PR first? 🤔 I think it should be fine now.

**Review by szattila98 — APPROVED** [review]
Sure, lets merge.

**Pull request merged**